platform_versioned_docs/version-24.2/monitoring/configtables/log_events.yml

```
@@ -15,6 +15,14 @@
   Resource:            'Data Explorer cloud buckets'
   Events logged:       'Add, edit, delete, hide, show'
```

Just noticed this - you can't `delete` a cloud bucket (!), but you can `remove` it from Data Explorer

```suggestion
  Events logged:       'Add, edit, remove, hide, show'
```